apps/courses/src/components/Header.tsx

```diff
@@ -278,11 +274,10 @@ export default ({ noScrolling }) => {
             />
           </RRDLink>
           <Box display={['block', null, 'none']} onClick={() => setShow(!show)}>
-            {/* SEE TODO (#3) */}
             {show ? (
-              <Icon as={FontAwesomeIcon} icon={faTimes} {...iconStyles} />
+              <Icon icon={faTimes} {...iconStyles} />
```

Comment:
Is `boxSize` defined anywhere? Or is it the default?

Reply:
The default is `4`. It is defined in the Icon component.